CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,25 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
+
         /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
+        * The Die class should contain one property to hold the current die value,
+        * and one method that rolls the die, returns and integer and takes no parameters.
          */
 
-        //Property
-
-
-        //Method
+        Random random = new Random();
 
+        private int rollIt;
+        public int RollIt
+        {
+            get { return rollIt; }
+            set { rollIt = value; }
+        }
 
+        public int Roll()
```

Comment:
+int Roll() is the method required by the brief, this meets the requirements. well done.